CTY-4_GHGemissions.ipynb

```diff
@@ -1,35 +1,22 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        'N2O': 265,
</pre><p>The methodology doc says <code>273</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wri/SCL-cities/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>